src/magic.cpp

```diff
@@ -1785,6 +1788,22 @@ void spellcasting_callback::draw_spell_info( const spell &sp, const uilist *menu
         }
         damage_string = string_format( "%s %d %s", _( "Summon" ), sp.damage(), _( monster_name ) );
         aoe_string = string_format( "%s: %d", _( "Spell Radius" ), sp.aoe() );
+    } else if( fx == "targeted_polymorph" ) {
+        std::string monster_name = sp.effect_data();
+        if( sp.has_flag( spell_flag::POLYMORPH_GROUP ) ) {
+            // TODO: Get a more user-friendly group name
+            if( MonsterGroupManager::isValidMonsterGroup( mongroup_id( sp.effect_data() ) ) ) {
+                monster_name = "random creature";
+            } else {
+                debugmsg( "Unknown monster group: %s", sp.effect_data() );
+            }
+        } else if( monster_name.empty() ) {
+            monster_name = "random creature";
+        } else {
+            monster_name = monster( mtype_id( sp.effect_data() ) ).get_name();
+        }
+        damage_string = string_format( "%s %dhp become a %s", _( "Targets under:" ), sp.damage(),
```

Comment:
The first string is not translated. Why are inserting a fixed string into another fixed string?
```suggestion
        damage_string = string_format( _( "Targets under: %dhp become a %s" ), sp.damage(),
```